pkg/operator/tpr.go

```diff
@@ -59,7 +59,7 @@ func (o *Operator) createTPR() error {
 		}
 	}
 
-	return err
```

Comment:
ahh i understand the code better.. you are not actually swalling the exception. You are returning all the errors except when it is already created. When i looked at it, i thought you were doing the opposite.